### PR TITLE
Extend flex data list with loading message, Ajax global option and improvement in fetching the initial value.

### PR DESCRIPTION
**Pull request by Klyner007**

### CODE REVIEW

File: jquery.flexdatalist.js [hunk 1]
```diff
@@ -15,1920 +15,1963 @@
...
-            // Option(s)
-            } else if (typeof _option === 'string') {
-                if (typeof _value === 'undefined') {
-                    return target.options.get(_option);
-                }
-                target.options.set(_option, _value);
-            }
-            return this;
+jQuery.flexdatalist = {};
+jQuery.flexdatalist.defaults = {
```
Comment (mac89) [inline jquery.flexdatalist.js]: Put `defaults` object directly in the object above.

File: jquery.flexdatalist.js [hunk 2]
```diff
@@ -15,1920 +15,1963 @@
...
+          $alias[ 0 ].value = "";
+        }
+        _selectedValues = [];
+        return this;
+      },
+      /**
+       * Value to object.
+       */
+      toObj: function( val ) {
+        if ( typeof val !
```
Comment (mac89) [inline jquery.flexdatalist.js]: Add as option and set default to true.

**Review by mac89 — COMMENTED** [review]
Add entry to the README.md mentioning the options that have been added.

**Comment by Klyner007** [comment]
Processed comments #1

**Comment by mac89** [comment]
README.md has not yet been updated.

**Comment by mac89** [comment]
OK

**Pull request merged**